package.json

```diff
@@ -50,7 +50,7 @@
     "prepublish": "npm run build"
   },
   "peerDependencies": {
-    "react-native": "0.67.0"
+    "react-native": "^0.67.1"
```

Comment:
Are we comfortable pickup any future 0.67.x automatically?

Reply:
I'm actually not sure how that happened.